openlibrary/plugins/upstream/utils.py

```diff
@@ -735,6 +732,8 @@ def get_donation_include(include):
 def item_image(image_path, default=None):
     if image_path is None:
         return default
+    if image_path[0:6] == 'https:':
```

Comment:
Not sure where this change is coming from, but done.

```suggestion
    if image_path.startswith('https:'):
```